src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1262,51 +1262,34 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
         }
 
         // Returns true if the status code is one of the supported redirection codes.
-        private static bool IsRedirectCode(HttpStatusCode statusCode)
+        private static bool IsRedirectCode(HttpStatusCode statusCode) => statusCode switch
         {
-            switch (statusCode)
-            {
-                case HttpStatusCode.Moved:
-                case HttpStatusCode.Found:
-                case HttpStatusCode.SeeOther:
-                case HttpStatusCode.TemporaryRedirect:
-                case HttpStatusCode.MultipleChoices:
-                case HttpStatusCode.PermanentRedirect:
-                    return true;
-                default:
-                    return false;
-            }
-        }
+            HttpStatusCode.Moved or
+            HttpStatusCode.Found or
+            HttpStatusCode.SeeOther or
+            HttpStatusCode.TemporaryRedirect or
+            HttpStatusCode.MultipleChoices or
+            HttpStatusCode.PermanentRedirect => true,
+            _ => false,
```

Comment:
I suggest sorting the elements. Below too.

Reply:
They are currently sorted in the same way as they are in System.Net.Http.SocketsHttpHandler.RedirectHandler.cs, are you sure I should sort them alphabetically?

Reply:
Yes, ABC order. If you want.